packages/coin-support-btc/src/index.ts

```diff
@@ -79,4 +82,12 @@ export class BtcSupport implements CoinSupport {
   public getAccountHistory(): Promise<IGetAccountHistoryResult> {
     throw new Error(`Method not implemented`);
   }
+
+  public getTransactionLink(txn: ITransaction): string {
+    return getTransactionLink(
+      coinList[txn.assetId] as IBtcCoinInfo,
+      txn.hash,
+      !!txn.confirmations,
+    );
+  }
```

Comment:
This already exists on `history` PR as `getExplorerLink`

Reply:
Merged history branch in this one